### DISCUSSION TITLE
如图把封面文字修改成特定的文字

### DISCUSSION THREAD

**Discussion by Yxiaofa** [nju-lug/NJUThesis#244] [Q&A]
将图一改成图二形式
![微信截图_20240314202814](https://github.com/nju-lug/NJUThesis/assets/98028423/240e3714-318e-4747-ac27-bcbaeff51332)
![微信截图_20240314202823](https://github.com/nju-lug/NJUThesis/assets/98028423/cae7cea3-e9e3-4b5b-b287-9cd8e4946908)

**Comment by atxy-blip** [comment]
您也许在找：`v1.2.0` 之前的旧版本，见 #204。不过既然研究生院已经在 2023 年启用了新版本封面，我觉得有必要找学院教务确认一下具体该用哪种。
